src/constants.ts

```diff
@@ -1 +1,33 @@
 export const nbsp = "\u00A0";
+
+export const DEFAULT_ERROR = "There was an error!";
+
+export const LOCALE = "en-GB";
+
+export const TABS = {
+    HOME: 0,
+    VIEW: 1,
+};
```

Comment:
`as const` or maybe an `enum` is better here.

Reply:
fixed